src/screens/DiscoveryScreenV2/index.js

```diff
@@ -26,7 +26,7 @@ import {Header} from '../../components';
 import {fonts} from '../../utils/fonts';
 
 const DiscoveryScreenV2 = ({route}) => {
-  const {tab} = route.params;
+  const {tab, initialTopics} = route.params;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `initialTopics` prop is being destructured from `route.params`. However, there's no check to ensure that `initialTopics` exists in `route.params`, which could potentially lead to an undefined value. It would be safer to provide a default value.
```diff
- const {tab, initialTopics} = route.params;
+ const {tab, initialTopics = []} = route.params;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->